hypothesis-python/tests/cover/test_example.py

```diff
@@ -87,12 +88,20 @@ def test_non_interactive_example_emits_warning():
 
 @pytest.mark.skipif(WINDOWS, reason="pexpect.spawn not supported on Windows")
 def test_interactive_example_does_not_emit_warning():
-    child = pexpect.spawn("%s -Werror" % (sys.executable,))
-    # If this test mysteriously fails here, it might be that your python
-    # can't launch cleanly with "-Werror". If you are running tests manually
-    # from a virtualenv, you might need to update your copy of virtualenv
-    # and create a fresh environment.
-    child.expect(">>> ", timeout=1)
+    try:
+        child = pexpect.spawn("%s -Werror" % (sys.executable,))
+        # If this test mysteriously fails here, it might be that your python
+        # can't launch cleanly with "-Werror". If you are running tests manually
+        # from a virtualenv, you might need to update your copy of virtualenv
+        # and create a fresh environment.
```

Comment:
This (existing) comment is now redundant with your new `except` block, which might be confusing.

So I think it makes sense to either rephrase the comment and move it down to just above the `skip` call, or just get rid of the comment entirely.

Reply:
If deleting the comment, the skip message will need to pick up the distinction between "a virtual environment" and "your installed virtualenv package", and suggest the fix.

Reply:
Do we know exactly which version of virtualenv fixed this bug?

Reply:
I think `16.2.0` based on when the fix was merged and the release dates, but I haven't actually validated this.

Reply:
Nope... it's not that important though, since the solution is to upgrade regardless and it's only a local skip in our test suite.  I'd be a lot more concerned if we were shipping something like this to users, but as a contributor experience thing IMO we've already overengineered it :smile: